src/fairchem/core/common/tutorial_utils.py

```diff
@@ -174,6 +174,9 @@ def nested_set(dic, keys, value):
         keys = _key.split(".")
         nested_set(config, keys, update[_key])
 
+    config["evaluation_metrics"]=config["eval_metrics"]
```

Comment:
would it be better to just use the updated key "evaluation_metrics" directly in the tutorial config, rather than correcting it here?

Reply:
I see - we should put a note on that and submit a fix in a separate PR that does not rename keys in utils vs base_trainer.

Reply:
@misko Can you add a TODO comment in the code about fixing this so that we do not forget?

Reply:
@lbluque Added this as an internal task with link to code and this PR